ddl.dic

```diff
@@ -546,6 +546,24 @@ save_dictionary.ddl_conformance
 
 save_
 
+save_dictionary.doi
+
+    _definition.id                '_dictionary.doi'
+    _definition.class             Attribute
+    _definition.update            2023-06-19
+    _description.text
+;
+    The digital object identifier (DOI) for this dictionary.
+;
+    _name.category_id             dictionary
+    _name.object_id               doi
+    _type.purpose                 Identify
+    _type.source                  Assigned
+    _type.container               Single
+    _type.contents                Word
```

Comment:
```suggestion
    _type.contents                Text
```

Reply:
We went with `Text` in all other DOI-related definitions. Probably because DOI strings can technically contain spaces ("DOI names may incorporate any printable characters, <...> which is the character set defined by Unicode) [1].

[1] https://www.doi.org/the-identifier/resources/handbook/2_numbering#2.5.1